crates/indexer/src/processors/token_processor.rs

```diff
@@ -226,7 +234,11 @@ fn insert_token_datas(
             diesel::insert_into(schema::token_datas::table)
                 .values(&token_datas_to_insert[start_ind..end_ind])
                 .on_conflict((token_data_id_hash, transaction_version))
-                .do_nothing(),
+                .do_update()
```

Comment:
is this for backfilling the data? 
should backfilling be done in a complete new run?

Reply:
Yea actually I already backfilled so technically I can set it back to do_nothing again lol.